compiler/rustc_middle/src/ty/mod.rs

```diff
@@ -118,6 +118,8 @@ mod sty;
 
 // Data types
 
+pub type RegisteredTools = FxHashSet<Ident>;
```

Comment:
It's used during name resolution, but in `ResolverOutputs` it could already be `FxHashSet<Symbol>`, I just didn't bother converting and used the same type everywhere.